app/views/groups/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render partial: 'shared/nav', locals: {title:'TRANSACTION', symbol:'←', url: :back}%>
+
+<section class="">
+  <p class="text-center mb-2"><%= notice %></p>
+  <%= render 'shared/detail', group:@group%>
+ 
+  <% @group.expenses.order(created_at: :desc).each do |g| %>
+    <div class="transact_card">
+    <h4>TRANSACTION NO: <%= g.id%> </h4>
+      <ul class="group_card_detail-inner">
+        <li class=""><%= g.name.capitalize %></li>
+        
+        <li>$<%= g.amount %></li>
+         </ul>
+         <div><%= g.created_at.strftime("%d %b %Y") %></div>
+    </div>
+   <% end %> 
+<%= button_to 'ADD TRANSACTION', new_user_group_expense_path(user_id: group.user.id, group_id: group.id), method: :get, class: 'category_button' %>
```

Comment:
- [ ] Like I mentioned in a previous comment before, please place the `ADD TRANSACTION` button at the bottom of the page as seen in the original design.
### Your Implementation
![image](https://github.com/Donmark2k/budget_app/assets/45460192/00216a70-2937-4962-9a5c-df3ea9364a50)
### Original Design
![image](https://github.com/Donmark2k/budget_app/assets/45460192/cd109230-3be2-4d4e-8a1f-315e71026e04)